assets/app/scripts/controllers/create/createFromImage.js

```diff
@@ -137,7 +142,14 @@ angular.module("openshiftConsole")
                         message: "Cannot create " + objectName + ". ",
                         details: failure.data.message
                       });
-                      hasErrors = true;
+                    }
+                  );
+                  result.success.forEach(
+                    function(success) {
+                      alerts.push({
+                        type: "success",
+                        message: "Created " + success.kind.toLowerCase() + " \"" + success.metadata.name + "\" successfully. "
```

Comment:
Based on @smarterclayton's previous statement about this, if we are seeing
error messages with un-humanized kinds we should be reporting those as
issues?

On Thu, Sep 24, 2015 at 2:21 PM, Fabiano Franz <notifications@github.com>
wrote:

> In assets/app/scripts/controllers/create/createFromImage.js
> <https://github.com/openshift/origin/pull/4728#discussion_r40353338>:
>
> > @@ -137,7 +142,14 @@ angular.module("openshiftConsole")
> >                          message: "Cannot create " + objectName + ". ",
> >                          details: failure.data.message
> >                        });
> > -                      hasErrors = true;
> > +                    }
> > +                  );
> > +                  result.success.forEach(
> > +                    function(success) {
> > +                      alerts.push({
> > +                        type: "success",
> > +                        message: "Created " + success.kind.toLowerCase() + " \"" + success.metadata.name + "\" successfully. "
>
> Fixed to humanize kind and objectName. Notice that eventually resource
> names can come as part of an api failure message, and we display it as it
> comes through failure.data.message. But for everything else we are
> covered.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4728/files#r40353338>.
>